packages/fxa-email-service/src/queues/mod.rs

```diff
@@ -36,7 +36,7 @@ pub struct Queues {
 }
 
 /// An incoming bounce/complaint queue.
-pub trait Incoming: Debug + Sync {
+pub trait Incoming: Debug + Send + Sync {
```

Comment:
I'm not sure which update caused it, but these need to be both `Send` and `Sync` now. Previously we compiled ok with just `Sync`.